codalab/worker/worker_run_state.py

```diff
@@ -255,7 +291,13 @@ def mount_dependency(dependency, shared_file_system):
         elif image_state.stage == DependencyStage.FAILED:
             # Failed to pull image; -> CLEANING_UP
             message = 'Failed to download Docker image: %s' % image_state.message
-            logger.error(message)
+            log_bundle_transition(
```

Comment:
Will this show a stacktrace? Maybe we should have a `show_stacktrace` parameter.

Reply:
keep the original code